mmhuman3d/core/visualization/visualize_smpl.py

```diff
@@ -580,8 +577,11 @@ def render_smpl(
             size of background images and finally resized to resolution.
         start (int, optional): start frame index. Defaults to 0.
 
-        end (int, optional): end frame index. Defaults to -1.
+        end (int, optional): end slice index. Excluded.
```

Comment:
I think `exclusive` is a clearer than `excluded`. And it is better to give an example.

Reply:
already solved. Our `end` serves the same as `list`, `tensor`, and `numpy` slice.